docs/knowledge_base/tips/long_callbacks/index.md

```diff
@@ -0,0 +1,73 @@
+# Long Running Callbacks
+
+Tasks in web applications can take a lot of time, which can lead to problems with communication between 
```

Comment:
I am concerned about the term "Task" here, which does not refer to the Taipy Task object. Can it be confusing for readers? @erikfTW @FabienLelaquais @FlorianJacta @RymMichaut

This comment applies to all the task usages on this page.

Alternative wording could be function, process, callback, etc.
What do you think?

Reply:
I completely agree.
We must find synonyms all over the place.
Here that could be replacing "Tasks in web applications" by "Executing a user-define piece of code"

Reply:
Agree! However, this is the terminology of the [doc](https://docs.taipy.io/en/develop/manuals/gui/callbacks/#long-running-callbacks). It should be changed there as well. The goal was to be consistent with the user man.

Reply:
A new ticket was created on taipy-doc for user manual:
https://github.com/Avaiga/taipy-doc/issues/603
We can proceed and apply the change on this page. We will take care of the user manual later.